### PR TITLE
feat: support silentmode to mute engine log

### PR DESCRIPTION
**Pull request by Duslia**
#### What type of PR is this?

feat
#### Check the PR title.

- [x] This PR title match the format: `<type>(optional scope): <description>`.
- [x] The description of this PR title is user-oriented and clear enough for others to understand.
- [x] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io).

#### (Optional) Translate the PR title into Chinese.
支持`安静模式`关闭 engine 的 log

#### (Optional) More detail description for this PR(en: English/zh: Chinese).

en: support silentmode to mute engine log
zh(optional): 支持`安静模式`关闭 engine 的 log

#### (Optional) The PR that updates user documentation:

https://github.com/cloudwego/cloudwego.github.io/pull/632

### CODE REVIEW

File: pkg/common/hlog/system.go [hunk 1]
```diff
@@ -23,6 +23,14 @@ import (
 	"sync"
 )
 
+var silentMode = false
+
+func SetSilentMode(s bool) {
```
Comment (welkeyever) [inline pkg/common/hlog/system.go:28]: 这个函数加点注释吧

File: pkg/common/hlog/system.go [hunk 2]
```diff
@@ -23,6 +23,14 @@ import (
...
+	silentMode = s
+}
+
+const MuteError = "Error=%s, remoteAddr=%s"
```
Comment (welkeyever) [inline pkg/common/hlog/system.go:32]: 小写够用了

File: pkg/common/hlog/system.go [hunk 3]
```diff
@@ -23,6 +23,17 @@ import (
...
+var silentMode = false
+
+// SetSilentMode is used to mute engine error log,
+// for example: error when reading request headers.
+// If true, hertz engine will mute it.
+func SetSilentMode(s bool) {
+	silentMode = s
+}
+
+const muteError = "Error=%s, remoteAddr=%s"
```
Comment (welkeyever) [inline pkg/common/hlog/system.go:35]: 挪到上面统一存放 var/ const 的地方吧

File: pkg/common/hlog/system.go [hunk 4]
```diff
@@ -23,6 +23,17 @@ import (
 	"sync"
 )
 
+var silentMode = false
```
Comment (welkeyever) [inline pkg/common/hlog/system.go:26]: 这个也是

Comment (Duslia) [reply pkg/common/hlog/system.go:28]: done

File: pkg/common/hlog/system.go [hunk 5]
```diff
@@ -23,6 +23,17 @@ import (
 	"sync"
 )
 
+const muteError = "Error=%s, remoteAddr=%s"
```
Comment (FGYFFFF) [inline pkg/common/hlog/system.go:26]: error 处理那块的  format是不是也可以换成这个

Comment (Duslia) [reply pkg/common/hlog/system.go]: done

**Review by welkeyever — DISMISSED** [review]

File: pkg/common/hlog/consts.go [hunk 6]
```diff
@@ -0,0 +1,7 @@
+package hlog
```
Comment (FGYFFFF) [inline pkg/common/hlog/consts.go:1]: 证书

Comment (Duslia) [reply pkg/common/hlog/consts.go:17]: done

**Review by welkeyever — APPROVED** [review]

**Review by wzekin — APPROVED** [review]

**Review by li-jin-gou — APPROVED** [review]

**Review by FGYFFFF — APPROVED** [review]

**Pull request merged**